### PR TITLE
Transform MVP pre-fill data to fit the all claims schema

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Takes the pre-fill data and manipulates it to match the all claims schema.

## Testing done
Unit test written.

Not sure how to test this better.

## Screenshots
N/A

## Acceptance criteria
- [x] The data that's output from `prefillTransformer` fits the schema for the all claims form

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx [hunk 1]
```diff
@@ -481,46 +484,73 @@ describe('526 helpers', () => {
...
-    const formData = {
-      'view:selectablePtsdTypes': {
-        'view:assaultPtsdType': true,
-      },
-      'view:upload781aChoice': 'answerQuestions',
-    };
-    expect(isAnswering781aQuestions(formData)).to.be.true;
+
+  describe('isAnswering781Questions', () => {
+    it('should return true if user has chosen to answer questions for a 781 PTSD incident', () => {
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx]: 👍

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 2]
```diff
@@ -138,6 +138,39 @@ export function transformDisabilities(disabilities = []) {
...
+    serviceInformation.servicePeriods = formData.servicePeriods;
+  }
+  if (formData.reservesNationalGuardService) {
+    serviceInformation.reservesNationalGuardService =
+      formData.reservesNationalGuardService;
+  }
+  const { servicePeriods, reservesNationalGuardService } = formData;
+  if (servicePeriods || reservesNationalGuardService) {
+    newFormData.serviceInformation = {
+      ..._.get('serviceInformation', newFormData, {}),
```
Comment (bshyong) [inline src/applications/disability-benefits/all-claims/utils.jsx]: `..._.`  morse code? 😂

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 3]
```diff
@@ -138,6 +138,39 @@ export function transformDisabilities(disabilities = []) {
   );
 }
 
+export function transformMVPData(formData) {
```
Comment (bshyong) [inline src/applications/disability-benefits/all-claims/utils.jsx]: LGTM to me and satisfies the transformation requirements in the associated issue
- Remove `veteran` and put it's properties as root-level properties
- Nest `servicePeriods` under `serviceInformation`
- Nest `reservesNationalGuardService` under `serviceInformation`

**Review by markgreenburg — APPROVED** [review]
LGTM, can get rid of that `serviceInformation` constant that isn't _really_ used anywhere.

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 4]
```diff
@@ -138,6 +138,39 @@ export function transformDisabilities(disabilities = []) {
...
+  );
+
+  // Spread the properties in formData.veteran
+  Object.keys(_.get('veteran', formData, {})).forEach(key => {
+    newFormData[key] = formData.veteran[key];
+  });
+
+  // Nest servicePeriods and reservesNationalGuardService under serviceInformation
+  //  without creating a serviceInformation property unnecessarily
+  const serviceInformation = {};
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/utils.jsx]: This guy's probably old, along with the following two conditionals

**Pull request merged**